### ISSUE TITLE
Dependency libx11-6:i386 depreciated in Ubuntu 64 bit

### ISSUE THREAD

**Issue by shivanshxyz** [LiquidGalaxyLAB/liquid-galaxy#14]
Issue:
Earlier, liquid galaxy installation required installation of libx11-6:i386 as a dependency. However, this package is now removed from the apt repositories of Ubuntu 64 bit. 

Solution:
replace libx11-6:i386 with libx11-6 seems to work fine as a replacement in 64 bit distros.

Tested on:

- [x] Ubuntu 16.04 LTS 64-bit
- [x] Ubuntu 20.04 LTS 64-bit

**Comment by Aju100** [comment]
Hi @shivanshxyz ,

what about 32bit distros who are running it ?

**Comment by shivanshxyz** [comment]
I believe it still works fine with Ubuntu 16.04 32 bit. But since 64 bit distros will be the way ahead from now on, it would be better in future to use 64 bit ubuntu

**Comment by Aju100** [comment]
@shivanshxyz  Yes I totally agree with your opinion, let the member of LG knows about it and assign the issue.

**Comment by gmpaliwal07** [comment]
Yes, I agree with you @shivanshxyz. I had to face the same issue when I am using Ubuntu's Latest version which is 22.04.
But in Ubuntu 16.04 it works Fine.
